game/Assets/Scripts/SentryRuntimeConfiguration.cs

```diff
@@ -14,6 +14,7 @@ public override void Configure(SentryUnityOptions options)
         // Take a look at `SentryBuildTimeOptionsConfiguration` instead.
         // TODO implement
 
+        options.AddInAppExcludeRegex(".*SentryTower.*"); // Sentry marks things started with 'Sentry' as InApp=false
```

Comment:
```suggestion
        options.AddInAppIncludeRegex(".*SentryTower.*"); // Sentry marks things started with 'Sentry' as InApp=false
```
I think we need to explicitly `include` this as `InApp`, right?